client/state/data-layer/wpcom/sites/memberships/index.js

```diff
@@ -39,10 +39,11 @@ export const membershipProductFromApi = ( product ) => ( {
 
 export const handleMembershipProductsList = dispatchRequest( {
 	fetch: ( action ) =>
+		//this is where you add more query params if necessary
 		http(
 			{
 				method: 'GET',
-				path: `/sites/${ action.siteId }/memberships/products`,
+				path: `/sites/${ action.siteId }/memberships/products?page=${ action.page }`,
```

Comment:
Does this affect the other uses of the memberships list, as mentioned by @mmtr in D49781 (queries that need all plans for dropdowns, etc)?

Reply:
I made the change on the backend to preserve the existing parameters so whatever consumer was using it before would have the same response returned to them. Good call!

Reply:
@kwight this doesn't affect how the plans are fetched in blocks, since they don't use this action:

https://github.com/Automattic/wp-calypso/blob/47d123dc0e9c102ca403bf91127c3914bb44c19f/apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/container/edit.js#L211

https://github.com/Automattic/jetpack/blob/3a551cbe3a56b93a89ed60a012059a865ee1fbf1/extensions/blocks/recurring-payments/edit.jsx#L95